### PR TITLE
refactor: visualizer and quick-access components

### PR DESCRIPTION
**Pull request by vik-13**
- Remove detaching app component. Now **zone.js** handles running change detection;
- Add **OnPush** strategy to most components;
- Fix issues with **quick-access** component (**fuse** search);
- Refactor **visualizer** component;
- Small improvements;

I know it's not necessary to have single module per component, especially for such small app. But as for me it makes component much more easier to understand what's going on there. I mean when you have one module for a few components, sometimes it's hard to say which imports are required for which component.

### CODE REVIEW

**Comment by mgechev** [comment]
Awesome refactoring! I'll take a look once I land ✈️

File: src/app/app.component.ts [hunk 1]
```diff
@@ -32,24 +32,13 @@ export class AppComponent implements AfterViewInit, OnDestroy {
...
-        this.loading = false;
-        return metadata;
-      })
-      .catch(() => (this.loading = false));
-  };
-
   private _stopLoading = () => {
     this.loading = false;
-    this._changeDetectorRef.detectChanges();
+    // this._changeDetectorRef.detectChanges();
```
Comment (mgechev) [inline src/app/app.component.ts:37]: Feel free to remove all the comments.

File: src/app/app.component.ts [hunk 2]
```diff
@@ -162,7 +149,7 @@ export class AppComponent implements AfterViewInit, OnDestroy {
   }
 
   get initialized() {
-    return this.manager.getCurrentState(() => this._changeDetectorRef.detectChanges());
+    return this.manager.getCurrentState(() => {});
```
Comment (mgechev) [inline src/app/app.component.ts:152]: Let us make the callback optional and remove it from here.

**Review by mgechev — APPROVED** [review]
LGTM! Left few minor suggestions.

**Comment by vik-13** [comment]
Updated

**Comment by mgechev** [comment]
Great job! That's a huge improvement compared to before. 🚀

**Comment by vik-13** [comment]
Thanks! I'm really looking forward when other views are gonna be implemented... It would be great to get a helpful app at the end :)

**Comment by mgechev** [comment]
Next I'll be working on restoring the previous component view we had with the template that allows you to navigate to directives & components used inside of it.

**Comment by vik-13** [comment]
Sounds great! I'm gonna create one PR soon. Found a solution to not reset the view in case of changing theme or other params. It's a bit tricky, but I'm testing it.. seems it works pretty smooth.

**Pull request merged**